### DISCUSSION TITLE
Issue with HTML5 audio tag and Range header

### DISCUSSION THREAD

**Discussion by pedromsfernandes** [tauri-apps/tauri#3185] [Q&A]
I am trying to port a simple music-player similar to Spotify from `electron-react-boilerplate` to `tauri` using `vite`. I have an API endpoint that streams the songs, and requires a range header to find the part of the song that it should return, and the `audio` tag should automatically handle this. Weirdly enough, if I open the app in the browser using the URL provided by `vite`, it still works, but it won't work with the `tauri` window, that is, the range header is not being sent. Is this a `webview` problem? Thanks in advance.

**Comment by amrbashir** [comment]
Please provide `tauri info`, code snippet of the audio tag with the link to the audio file you are trying to play.
